source/php/Component/Iframe/iframe.blade.php

```diff
@@ -2,10 +2,6 @@
 <iframe 
     id="{{ $id }}" 
     class="{{ $class }}" 
-    data-src="{{ $src }}" 
-    loading="{{ $loading }}" 
-    height="{{ $height }}" 
-    width="{{ $width }}"
-    frameborder="0" 
-    src="{{ $src }}">
+	options="{{ $options }}"
```

Comment:
Not a valid attribute?

Reply:
Having issues with double quotation marks in the json string that $options contains being formatted incorrectly when being parsed by [buildAttributes](https://github.com/helsingborg-stad/component-library/blob/b613c0b6923a67f638b4bc581b6cf2be0c19136f/source/php/Component/BaseController.php#L301-L313) so I put it in like this temporarily.